web/package.json

```diff
@@ -19,15 +19,15 @@
     "lint:style": "stylelint --fix \"src/**/*.less\" --syntax less",
     "prettier": "prettier -c --write \"**/*\"",
     "site": "yarn run fetch:blocks && yarn run build",
-    "start": "umi dev",
-    "start:e2e": "cross-env SERVE_ENV=test umi dev",
+    "start": "cp -R ./node_modules/monaco-editor ./public/ && umi dev",
```

Comment:
To load external sources from local.related issue is [here](https://github.com/suren-atoyan/monaco-react/issues/168).

Reply:
Got it, thanks for your instructions. While our dashboard probably connected to the Internet in most scenes, how about use CDN way by default(if it is stable), that will reduce the dependence , and we can offer this way in our FAQ, what do you think?

Reply:
But in order to allow users to not use CDN, we still need to copy the monaco code to the `public` directory, which will not reduce the dependency.

Reply:
I noticed that 
> NOTE: For TypeScript type definitions, this package uses the monaco-editor package as a peer dependency (reference: https://github.com/suren-atoyan/monaco-react)

so yep, the `monaco-editor` dependence can not be reduced no matter what way we use 

hope a neater way to do this instead of cope `monaco` to the public dir